source/slang/core.meta.slang.h

```diff
@@ -1140,6 +1140,11 @@ SLANG_RAW("\n")
 SLANG_RAW("__attributeTarget(DeclBase)\n")
 SLANG_RAW("attribute_syntax [gl_binding(binding: int, set: int = 0)]\t\t\t: GLSLBindingAttribute;\n")
 SLANG_RAW("\n")
+SLANG_RAW("__attributeTarget(DeclBase)\n")
+SLANG_RAW("attribute_syntax [vk_push_constant]\t\t\t\t\t\t\t\t\t: PushConstantAttribute;\n")
```

Comment:
Hmm... All these `\t`s here make me realize that there are a bunch of tab characters in `core.meta.slang`. We should probably clean those up, but I'm not too concerned since it isn't a functionality issue.